client/clientmanager/src/main/java/io/mosip/registration/clientmanager/repository/GlobalParamRepository.java

```diff
@@ -3,26 +3,70 @@
 import io.mosip.registration.clientmanager.dao.GlobalParamDao;
 import io.mosip.registration.clientmanager.entity.GlobalParam;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 import javax.inject.Inject;
 
 public class GlobalParamRepository {
 
-    private GlobalParamDao globalParamDao;
+    private static final String TAG = GlobalParamRepository.class.getSimpleName();
+
+    private static Map<String, String> globalParamList;
+
+    private static GlobalParamDao globalParamDao;
 
     @Inject
     public GlobalParamRepository(GlobalParamDao globalParamDao) {
-        this.globalParamDao = globalParamDao;
+        GlobalParamRepository.globalParamDao = globalParamDao;
     }
 
     public String getGlobalParamValue(String id) {
-        return this.globalParamDao.getGlobalParam(id);
+        return globalParamDao.getGlobalParam(id);
     }
 
     public void saveGlobalParam(String id, String value) {
         GlobalParam globalParam = new GlobalParam(id);
         globalParam.setName(id);
         globalParam.setValue(value);
         globalParam.setStatus(true);
-        this.globalParamDao.insertGlobalParam(globalParam);
+        globalParamDao.insertGlobalParam(globalParam);
+        refreshGlobalParams();
```

Comment:
we don't have to refresh complete static list here, we can directly call below statement
globalParamList.put(globalParam.getId(), globalParam.getValue());

Reply:
done